### PR TITLE
fix(src/views/newUserOnboarding/index) unintended untype

### PR DESCRIPTION
**Pull request by ryota-murakami**
**Status**
- [x] Ready for review

**Deploy after merge (delete what needn't be deployed)**
- hyperion (frontend)

[src/views/newUserOnboarding/index.js](https://github.com/withspectrum/spectrum/blob/alpha/src/views/newUserOnboarding/index.js) is overall flow-typed, but there is no `@// flow`:sweat_smile:

### CODE REVIEW

File: src/views/newUserOnboarding/index.js [hunk 1]
```diff
@@ -21,13 +20,27 @@ import {
   StickyRow,
   ContinueButton,
 } from './style';
+import type { UserInfoType } from '../../../shared/graphql/fragments/user/userInfo';
```
Comment (brianlovin) [inline src/views/newUserOnboarding/index.js]: You can import directly from `shared/..` - no need for relative paths :)

Comment (ryota-murakami) [reply src/views/newUserOnboarding/index.js]: @brianlovin thanks, i fixed it!
In relation to this, flow type information is valuable to know **what kind of data?** it deals with people who are not familiar with codebase like me, so I'd like to continue to contribute!:smile: (cc: @mxstbr )

e.g. Replacing a simple `Object` type to a more appropriate GraqhQL Fragment types.

Comment (mxstbr) [reply src/views/newUserOnboarding/index.js]: That sounds great! We'd love contributions to improve the flow types of the codebase :100:

**Review by mxstbr — APPROVED** [review]
LGTM!

File: src/views/newUserOnboarding/index.js (hunk 1 above)
Comment (ryota-murakami) [reply src/views/newUserOnboarding/index.js]: @mxstbr thanks bunch of review, and appreciate.
i think flow is not affect in runtime code, so we can improve aggressively when you thought it!

i like James Kyle's thought, direction, so i reminder myself to not become **dogmatic typeckeck man** , for more human friendly information😀
https://discuss.reactjs.org/t/if-typescript-is-so-great-how-come-all-notable-reactjs-projects-use-babel/4887/2

also flow provide `// @noflow` escape hatch(also you know `$FlowFixMe`), we can use these 
as well for do not sacrifice productivity👩‍💻
https://github.com/facebook/flow/issues/1321#issuecomment-181541953

Comment (mxstbr) [reply src/views/newUserOnboarding/index.js]: Yep, you get it—that's exactly how we use it! :+1: Any improvements to our flow types is very welcome.

**Pull request merged**